modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -236,6 +236,7 @@ Therefore, use it with care, especially with large imports.
 * The first line must contain the header.
 * Multiple data sources like these can be specified in one import, where each data source has its own header.
 * Files can also be specified using regular expressions.
+* As of Neo4j 5.19, files can also be located in S3 URIs.
```

Comment:
And here

`neo4j-admin database import full --nodes s3://my-bucket/data/large.csv -- newdb`

Reply:
added a note, an example and included info about AWS default credentials